package.json

```diff
@@ -38,6 +38,7 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "@zowe/cli": "^7.16.6",
```

Comment:
Please remove, sorry that I missed this before. We don't want the CLI to be a dependency of itself 😋